python/ray/serve/context.py

```diff
@@ -151,3 +151,8 @@ class RequestContext:
 _serve_request_context = contextvars.ContextVar(
     "Serve internal request context variable", default=RequestContext()
 )
+
+
+# Context Information of replica actor.
+REPLICA_DEPLOYMENT_NAME = ""
+REPLICA_TAG_NAME = ""
```

Comment:
these are already in the `ReplicaContext` that's set above in `_set_internal_replica_context`, please unify on that

Reply:
nice catch!